modules/core/arrow-extras-data/src/main/kotlin/arrow/data/StateT.kt

```diff
@@ -76,7 +77,7 @@ class StateT<F, S, A>(
      * @param fa the value to liftF.
      */
     fun <F, S, A> liftF(AF: Applicative<F>, fa: Kind<F, A>): StateT<F, S, A> = AF.run {
-      StateT(just({ s -> fa.map { a -> Tuple2(s, a) } }))
+      StateT(just { s -> fa.map { a -> Tuple2(s, a) } })
```

Comment:
...this looks wrong. The lambda is a parameter.

Reply:
that was what kotlin suggested.

Reply:
I believe that.

Reply:
So, should I revert the change?

Reply:
When lambdas are single parameter you can omit the braces and just inline the function. StateT wraps a function.